lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -431,6 +431,9 @@ estimateMaxInputsTests cases = do
         forM_ cases $ \(GivenNumOutputs nOuts, ExpectedNumInputs nInps) -> do
             let (o,i) = (show nOuts, show nInps)
             it ("order of magnitude, nOuts = " <> o <> " => nInps = " <> i) $ do
+                -- NOTE: These tests broke in the GHC 8.6 -> 8.10 bump,
+                -- presumably due to some change in the arbitrary generation.
+                -- It would be better if they weren't so fragile.
```

Comment:
We could perhaps "dump" the generated values in files used as input data for the test?

Reply:
Probably - don't think it'd be worth the effort though. Would be 50 outputs in the source file.

But I wonder if this wouldn't be nice to have some day:
- `50 arbitrary outputs -> at least X1 inputs, never more than X2 inputs`
- `50 fixed-size outputs -> always Y inputs`

i.e. properties rather than goldens.